src/Main.cpp

```diff
@@ -0,0 +1,20 @@
+/**
+ * @mainpage
+ * @file
+ * @brief Start point
+ * @author LionZXY
+ *
+ * Start point Recast server. Initializing @see Server and @see MainThread.
+ * Init config class @see Config
+ **/
+#include <Server.h>
+#include "stdio.h"
```

Comment:
For systems headers you should use `<>` instead `""`. And `""` for your own headers.